loki-logback-appender/src/test/java/com/github/loki4j/logback/JavaHttpAppenderTest.java

```diff
@@ -47,7 +47,20 @@ public void testJavaHttpSend() {
 
             a.appendAndWait(events[2]);
             assertEquals("http send", expected, new String(mockLoki.lastBatch));
+            assertTrue(mockLoki.tenantHeaders.isEmpty());
+            return null;
+        });
+    }
+
+    @Test
+    public void testJavaHttpSendWithTenantHeader() {
```

Comment:
To make sure the whole this is working we need an integration test. Check [integration tests](https://github.com/loki4j/loki-logback-appender/tree/main/loki-logback-appender/src/test/java/com/github/loki4j/logback/integration) and especially [auth test](https://github.com/loki4j/loki-logback-appender/blob/main/loki-logback-appender/src/test/java/com/github/loki4j/logback/integration/GrafanaCloudTest.java)  for examples.

Reply:
We can drop this test in favor of integration test

Reply:
I would like to keep it as unit test also. Just in case if someone can not run integration tests during development cycle